### PR TITLE
Аdd unified runner

### PR DESCRIPTION
**Pull request by leshikus**
@pbchekin could you please take a look before I start switching to new runners?

### CODE REVIEW

File: .github/dockerfiles/runner-base/Dockerfile [hunk 1]
```diff
@@ -49,3 +49,8 @@ RUN \
   cd $HOME; \
   /bin/sh l_BaseKit*.sh -a --silent --eula accept; \
   rm l_BaseKit*.sh
+
+RUN \
+  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; \
+  sh Miniconda3-latest-Linux-x86_64.sh -b
```
Comment (pbchekin) [inline .github/dockerfiles/runner-base/Dockerfile:55]: WORKDIR is not set, / is not writable by user runner. Also see this hadolint warning https://github.com/hadolint/hadolint/wiki/DL4001 (Either use wget or curl but not both).
```suggestion
RUN \
  cd /tmp; \
  curl -sSLO https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; \
  sh Miniconda3-latest-Linux-x86_64.sh -b; \
  rm Miniconda3-latest-Linux-x86_64.sh
```

File: .github/dockerfiles/runner-base/Dockerfile [hunk 2]
```diff
@@ -39,13 +38,18 @@ RUN set -ex; \
...
 # TODO: install only necessary components
-
 RUN \
-  cd $HOME; \
   /bin/sh l_BaseKit*.sh -a --silent --eula accept; \
   rm l_BaseKit*.sh
+
+RUN \
+  curl -sSLO https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; \
+  sh Miniconda3-latest-Linux-x86_64.sh -b
```
Comment (pbchekin) [inline .github/dockerfiles/runner-base/Dockerfile:54]: Why do you keep `.sh` file in the image?

File: .github/dockerfiles/runner-base/Dockerfile [hunk 3]
```diff
@@ -36,16 +36,22 @@ RUN set -ex; \
...
 # TODO: install only necessary components
-
 RUN \
-  cd $HOME; \
   /bin/sh l_BaseKit*.sh -a --silent --eula accept; \
   rm l_BaseKit*.sh
+
+RUN \
+  curl -sSLO https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; \
+  sh Miniconda3-latest-Linux-x86_64.sh -b;
```
Comment (pbchekin) [inline .github/dockerfiles/runner-base/Dockerfile:56]:
```suggestion
  sh Miniconda3-latest-Linux-x86_64.sh -b; \
```

**Comment by leshikus** [comment]
Close temporarily

**Review by pbchekin — APPROVED** [review]
Looks good!

File: .github/dockerfiles/runner-base/Dockerfile (hunk 3 above)
Comment (leshikus) [reply .github/dockerfiles/runner-base/Dockerfile]: Fixed, thanks!

**Pull request merged**